spec/api-content-tracing-spec.ts

```diff
@@ -112,7 +112,8 @@ ifdescribe(!(['arm', 'arm64'].includes(process.arch)) || (process.platform !== '
       expect(fs.statSync(path).isFile()).to.be.true('output exists');
     });
 
-    it('calls its callback with a result file path', async () => {
+    // FIXME(ckerr): this test regularly flakes
+    it.skip('calls its callback with a result file path', async () => {
```

Comment:
followup issue: https://github.com/electron/electron/issues/45579

Reply:
```suggestion
    ifit(process.platform !== 'linux')('calls its callback with a result file path', async () => {
```

Reply:
We should only disable test for Linux. (You also will have to add the ifit helper to this test).